widget/form_test.go

```diff
@@ -132,3 +133,35 @@ func TestForm_ChangeTheme(t *testing.T) {
 		test.AssertImageMatches(t, "form/theme_changed.png", w.Canvas().Capture())
 	})
 }
+
+func TestForm_Validation(t *testing.T) {
+	app := test.NewApp()
+	defer test.NewApp()
+	app.Settings().SetTheme(theme.LightTheme())
+
+	entry1 := &Entry{Validator: validation.NewRegexp(`^\d{2}-\w{4}$`, "Input is not valid"), Text: "15-true"}
+	entry2 := &Entry{Validator: validation.NewRegexp(`^\w{3}-\w{5}$`, "Input is not valid"), Text: "wrong"}
+	entry3 := &Entry{}
+	items := []*FormItem{
+		{Text: "First", Widget: entry1},
+		{Text: "Second", Widget: entry2},
+		{Text: "Third", Widget: entry3},
+	}
+
+	form := &Form{Items: items, OnSubmit: func() {}, OnCancel: func() {}}
+	w := test.NewWindow(form)
+	defer w.Close()
+
+	test.AssertImageMatches(t, "form/validation_initial.png", w.Canvas().Capture())
+
+	test.Type(entry2, "not-")
+	entry1.SetText("incorrect")
+	w = test.NewWindow(form)
+
+	test.AssertImageMatches(t, "form/validation_invalid.png", w.Canvas().Capture())
+
+	entry1.SetText("15-true")
+	w = test.NewWindow(form)
+
+	test.AssertImageMatches(t, "form/validation_valid.png", w.Canvas().Capture())
```

Comment:
I don't quite get why this image has the submit button icon not showing.

Reply:
It may relate to an issue regarding disabled buttons.
It may just relate to it.
Can you try the same code based on develop and see if you have the same issue?

Reply:
Same issue sadly. I have no idea as to what is causing it.